.github/workflows/python.yml

```diff
@@ -40,9 +40,9 @@ jobs:
           pip install --no-cache-dir torch==1.5.0
           pip install --no-cache-dir ./submarine-sdk/pysubmarine/.
           pip install -r ./submarine-sdk/pysubmarine/github-actions/test-requirements.txt
-          pip install -r ./submarine-sdk/pysubmarine/github-actions/lint-requirements.txt
+          pip install -r ./dev-support/style-check/python/lint-requirements.txt
       - name: Check python sdk code style
-        run: ./submarine-sdk/pysubmarine/github-actions/lint.sh
+        run: ./dev-support/style-check/python/lint.sh
```

Comment:
Could we remove `lint-requirements.txt` and `lint.sh` in `./submarine-sdk/pysubmarine/github-actions/`

Reply:
OK. I will remove it and also remove the `auto-format.sh` in `./submarine-sdk/pysubmarine/github-actions/`.